lucene/analysis/kuromoji/src/test/org/apache/lucene/analysis/ja/dict/UserDictionaryTest.java

```diff
@@ -77,4 +78,11 @@ public void testRead() throws IOException {
     UserDictionary dictionary = TestJapaneseTokenizer.readDict();
     assertNotNull(dictionary);
   }
+
+  @Test(expected = RuntimeException.class)
```

Comment:
Can you use `expectThrows` and check that the message is correct ?

Reply:
Thanks, fixed.